PlayKitSmartSwitch/Classes/SmartSwitchMediaEntryInterceptor.swift

```diff
@@ -72,10 +72,24 @@ extension SmartSwitchMediaEntryInterceptor: PKMediaEntryInterceptor {
         }
     }
     
-    private func getOrderedCDN(originalURL: URL, completion: @escaping (_ cdn: SmartSwitcCDNItem?, _ error: Error?) -> Void) {
-        guard let request: KalturaRequestBuilder = KalturaRequestBuilder(url: self.config.smartSwitchUrl,
+    private func getOrderedCDN(originalURL: URL,
+                               completion: @escaping (_ cdn: Provider?, _ error: Error?) -> Void) {
+        
+        var serverURL = self.config.domainUrl
+        
+        serverURL = serverURL.replacingOccurrences(of: "{accountCode}", with: self.config.accountCode)
```

Comment:
If the accountCode was not set, it will replace it with an empty string, is that ok?
Maybe there should be a default one?

Reply:
accountCode it is required parameter if it will be missing you will have error message after API call.
400 "Invalid account code"

Reply:
Ok, so what I understand, it is up to the client to set the accountCode in the config.

Reply:
@chausov Don't we want to return an error before we go on and create the request and get the error from the API call? Or should we let the server decided?

Reply:
it is up to the client to set the account code and also this account code should be correct. Same as for Youbora we did not checking if account code is correct.

Reply:
it is up to the client to set the account code and also this account code should be correct. Same as for Youbora we did not checking if account code is correct.